lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -236,7 +213,7 @@ describe('BasicLtiLaunchApp', () => {
     fakeConfig.submissionParams = undefined;
 
     renderLtiLaunchApp();
-    await nextTick();
+    await new Promise(resolve => setTimeout(resolve, 0));
```

Comment:
This code does the same as the previous `await nextTick()`, but I removed that helper since it was only used in one place.